go.mod

```diff
@@ -4,24 +4,26 @@ go 1.17
 
 require (
 	github.com/go-logr/logr v1.2.0
+	github.com/jackc/pgx/v4 v4.15.0
 	github.com/lib/pq v1.10.4
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.17.0
+	github.com/spf13/viper v1.10.1
```

Comment:
Hmm: do we _need_ to add viper as a direct dependency here? Can we instead use the cobra CLI library entirely, which  IIRC uses viper under-the-hood, and we can drop viper as a direct dependency entirely?